pkg/asset/ignition/machine/node.go

```diff
@@ -5,7 +5,8 @@ import (
 	"net"
 	"net/url"
 
-	ignition "github.com/coreos/ignition/config/v2_2/types"
+	"github.com/coreos/ignition/v2/config/util"
+	igntypes "github.com/coreos/ignition/v2/config/v3_1/types"
```

Comment:
Keep `ignition` to minimize the diff?

Reply:
If it's not too much of a pain reviewing, I'd rather import it as `igntypes` like everywhere else in this repo.
This is @abhinavdahiya's call :)